sdk/freeport/freeport.go

```diff
@@ -125,6 +128,12 @@ func reset() {
 		lockLn = nil
 	}
 
+	if ticker != nil {
+		killTicker <- true
+		ticker.Stop()
```

Comment:
```suggestion
		ticker.Stop()
		killTicker <- true
```

I think it has to be this way around. Otherwise it would be possible that after reading from killTicker, to enter the loop again and read from the ticker.

Reply:
I think it is not as `checkFreedPortsOnce()` is protected by `mu.Lock()`, so from the grouting perspective, `ticker.Stop()` and `killTicker <- true` are atomic, but I am doing your suggestion anyway because it is easier to read anyway

Reply:
@i0rek Fixed in next commit